templates/comments_add_visitor.html

```diff
@@ -35,7 +35,7 @@
     -->
 </script>
 
-<hr name="addcomment">
+<hr id="addcomment">
 <form id="post" name="post" action="comments.php" method="post" onsubmit="return chkFormular()" class="form-horizontal">
```

Comment:
We may achieve the same effect by making the header elements themselves the anchors:

```xml
<H1>Table of Contents</H1>
<P><A href="#section1">Introduction</A><BR>
...the rest of the table of contents...
...the document body...
<H2 id="section1">Introduction</H2>
...section 1...
```
http://www.w3.org/TR/html401/struct/links.html#h-12.1.1 - the <hr>-Tag does not have the name-Attribute.